feature/experimental/tunnel/otg_tests/tunnel_loadbalance_acl_based/tun_acl_loadbalance_test.go

```diff
@@ -0,0 +1,868 @@
+
+	s6 := s.GetOrCreateIpv6()
+	if deviations.InterfaceEnabled(tc.dut) {
+		s6.Enabled = ygot.Bool(true)
+	}
+	s6.GetOrCreateAddress(a.IPv6).PrefixLength = ygot.Uint8(plen6)
+}
+
+func (tc *testCase) configDstMemberDUT(i *oc.Interface, p *ondatra.Port) {
+	i.Description = ygot.String(p.String())
+	i.Type = ethernetCsmacd
+	if deviations.InterfaceEnabled(tc.dut) {
+		i.Enabled = ygot.Bool(true)
+	}
+
+	e := i.GetOrCreateEthernet()
+	e.AggregateId = ygot.String(tc.aggID)
+}
+
+func (tc *testCase) configureDUT(t *testing.T) {
+	t.Logf("dut ports = %v", tc.dutPorts)
+	if len(tc.dutPorts) < 2 {
+		t.Fatalf("Testbed requires at least 2 ports, got %d", len(tc.dutPorts))
+	}
+	d := gnmi.OC()
+	tc.clearAggregateMembers(t)
+	tc.setupAggregateAtomically(t)
+	if tc.lagType == lagTypeLACP {
+		lacp := &oc.Lacp_Interface{Name: ygot.String(tc.aggID)}
+		lacp.LacpMode = oc.Lacp_LacpActivityType_ACTIVE
+
+		lacpPath := d.Lacp().Interface(tc.aggID)
+		fptest.LogQuery(t, "LACP", lacpPath.Config(), lacp)
+		gnmi.Replace(t, tc.dut, lacpPath.Config(), lacp)
+	}
+	// TODO - to remove this sleep later
+	time.Sleep(5 * time.Second)
+
+	agg := &oc.Interface{Name: ygot.String(tc.aggID)}
+	tc.configDstAggregateDUT(agg, &dutDst)
+	aggPath := d.Interface(tc.aggID)
+	fptest.LogQuery(t, tc.aggID, aggPath.Config(), agg)
+	gnmi.Replace(t, tc.dut, aggPath.Config(), agg)
+
+	srcp := tc.dutPorts[0]
+	srci := &oc.Interface{Name: ygot.String(srcp.Name())}
+	tc.configSrcDUT(srci, &dutSrc)
+	srci.Type = ethernetCsmacd
+	srciPath := d.Interface(srcp.Name())
+	fptest.LogQuery(t, srcp.String(), srciPath.Config(), srci)
+	gnmi.Replace(t, tc.dut, srciPath.Config(), srci)
+	if deviations.ExplicitInterfaceInDefaultVRF(tc.dut) {
+		fptest.AssignToNetworkInstance(t, tc.dut, srcp.Name(), deviations.DefaultNetworkInstance(tc.dut), 0)
+		fptest.AssignToNetworkInstance(t, tc.dut, tc.aggID, deviations.DefaultNetworkInstance(tc.dut), 0)
+	}
+	for _, port := range tc.dutPorts[1:] {
+		i := &oc.Interface{Name: ygot.String(port.Name())}
+		tc.configDstMemberDUT(i, port)
+		iPath := d.Interface(port.Name())
+		fptest.LogQuery(t, port.String(), iPath.Config(), i)
+		gnmi.Replace(t, tc.dut, iPath.Config(), i)
+	}
+	if deviations.ExplicitPortSpeed(tc.dut) {
+		for _, port := range tc.dutPorts {
+			fptest.SetPortSpeed(t, port)
+		}
+	}
+	configureNetworkInstance(t)
+	gnmiClient := tc.dut.RawAPIs().GNMI(t)
+	var v4Config, v6Config string
+	if deviations.TunnelAclEncapsulationConfigUnsupported(tc.dut) {
+		switch tc.dut.Vendor() {
+		case ondatra.JUNIPER:
+			v4Config = juniperEncapCLI("inet", aclIpv4Name, encapSrcMatch, encapDstMatch, countIpv4, tunnelActionIpv4, tunnelEndpoint, "ipv4", greSrcAddr, greDstAddr)
+			v6Config = juniperEncapCLI("inet6", aclIpv6Name, encapV6SrcMatch, encapv6DstMatch, countIpv6, tunnelActionIpv6, tunnelV6Endpoint, "ipv6", greV6SrcAddr, greV6DstAddr)
+			t.Logf(" Push CLI config of:\n%s", tc.dut.Vendor())
+		default:
+			t.Errorf("Invalid Filter configuration")
```

Comment:
Make err msg more specific like

t.Errorf("Filter configuration is not defined for vendor %s",  tc.dut.Vendor())

Reply:
thanks for the review added t.Errorf("Vendor %s specific acl based tunnel configuration is not defined", tc.dut.Vendor()) now.